x/campaign/types/params.go

```diff
@@ -12,12 +12,14 @@ import (
 )
 
 var (
-	DefaultMinTotalSupply      = sdkmath.NewInt(100)                   // One hundred
-	DefaultMaxTotalSupply      = sdkmath.NewInt(1_000_000_000_000_000) // One Quadrillion
-	DefaultCampaignCreationFee = sdk.Coins(nil)                        // EmptyCoins
+	DefaultMinTotalSupply             = sdkmath.NewInt(100)                   // One hundred
+	DefaultMaxTotalSupply             = sdkmath.NewInt(1_000_000_000_000_000) // One Quadrillion
+	DefaultCampaignCreationFee        = sdk.Coins(nil)                        // EmptyCoins
+	DefaultMaxMetadataLength   uint64 = 1000
```

Comment:
Can we actually increase it to `2000`?
Since we have more metadata we're considering for the launchpad